apps/testingaccessibility/src/components/content/new-landing-copy.mdx

```diff
@@ -188,15 +188,15 @@ which helps me when I start thinking about how I'm going to solve accessibility
 
 </Section>
 
-<Section className="bg-moss-50 prose-green" image={require('../../../public/assets/lighthouse@2x.png')}>
+<Section aria-label="Accessibility in the design phase" className="bg-moss-50 prose-green" image={require('../../../public/assets/lighthouse@2x.png')}>
```

Comment:
Looks like these aria-labels aren't making it through as props.

Reply:
will fix!

Reply:
I fixed it in the PR!